src/tools/dartpad.md

```diff
@@ -168,3 +168,9 @@ To create a simple web app, start with the Hello World HTML sample.
     </ol>
   </li>
 </ol>
+
+### Checking DartPad version info
```

Comment:
Hmm, not sure what you mean by _"..make this a command"_  + why not set it to _"Check the **DartPad's** version"_ it would be easier to find when searching.